src/test/java/com/hillel/items_exchange/controller/ImageControllerIntegrationTest.java

```diff
@@ -69,16 +67,6 @@ void getImageLinksByProductId_shouldReturnAllImageLinks() throws Exception {
                 .andExpect(jsonPath("$[1].defaultPhoto").value(true));
     }
 
-    @WithMockUser("admin")
-    @Test
-    @ExpectedDataSet(value = "image/create.yml", ignoreCols = {"created", "updated"})
-    void saveImages_shouldWriteNewImageWhenSupportedType() throws Exception {
-        mockMvc.perform(multipart("/image/{product_id}", 2L)
-                .file(jpeg).file(png))
-                .andDo(print())
-                .andExpect(status().isOk());
-    }
-
```

Comment:
Please, could you explain why you deleted this test case?

Reply:
Mocked controller couldn't work with real images. So I had to remove test for that here and to create new ones into ImageServiceTest.